custom_student_verification/compat.py

```diff
@@ -67,3 +67,25 @@ def get_base_message_type():
     except ImportError:
         from edx_ace.message import MessageType
         return MessageType
+
+
+def get_course_enrollment_model():
+    """
+    Get CourseEnrollment model.
+    """
+    try:
+        from common.djangoapps.student.models import CourseEnrollment
+        return CourseEnrollment
+    except ImportError:
+        return object
```

Comment:
@farhaanbukhsh what advantage do we have here if we return an `object` instead of throwing error?

Reply:
It helps to run the test without failing and we can use this functions as mock if we want to write tests.